lib/rules/function-no-unknown/index.js

```diff
@@ -43,7 +43,16 @@ const rule = (primary, secondaryOptions) => {
 			return;
 		}
 
-		const functionsList = JSON.parse(fs.readFileSync(functionsListPath.toString(), 'utf8'));
+		const functionsList = [
+			...JSON.parse(fs.readFileSync(functionsListPath.toString(), 'utf8')),
+			// #5960
+			'-webkit-gradient',
+			'color-stop',
+			'from',
+			'to',
```

Comment:
[ask] Adding `-webkit-gradient` seems reasonable because [`css-function-list` includes vendor-prefixed functions](https://github.com/niksy/css-functions-list/blob/0c17ffe6f4fcc43afb4dcf4befca3ceb7dd23b6b/index.json#L122). However, adding `color-stop`, `from`, and `to` may cause false negatives. Are they described on any specs?

Reply:
They are part of the functions used by `-webkit-gradient` value.
check https://github.com/stylelint/stylelint/pull/6539/files#diff-ec67c2191547de2436754f5cc78f9a4f61564730f4215b69cd3829a0d58af7b6R26

Reply:
Understood. But I'm still concerned about potential false negatives. The following report would not be emitted if this change would be merged:

```css
a { background: color-stop(...); }
             /* ↑ Unexpected unknown function "color-stop" */
```

Reply:
AFAIK we have the same problem with all functions that required nesting.

e.g.
```
width: cos(pi); // invalid (but passes)
width: calc(100px * cos(pi)); // valid
```

The rule name is `function-no-unknown` so I consider the problem a non-issue.

Reply:
That makes sense. 👍🏼